### PR TITLE
chore: add .gitattributes file

### PR DESCRIPTION
**Pull request by tirelyl**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Enforce Unix newlines
+*.*     text eol=lf
```
Comment (wendellhu95) [inline .gitattributes]: 这个不是覆盖了下面所有的文件类型吗？

**Comment by tirelyl** [comment]
If you already have an existing repository, you can preform the next commands to autofix the line-break style problem.
First you should make sure that your local repo is clean and has nothing to commit. then run

```bash
# This will force git to recheck and "reapply" gitattributes changes.
git rm --cached -r .
git reset --hard
```

File: .gitattributes (hunk 1 above)
Comment (tirelyl) [reply .gitattributes]: 好像是的，我看的一个别人的库里面的 。😅 我查找了一下资料，有一个 github 库收集了各种 gitattributes 的模板 https://github.com/alexkaratarakis/gitattributes ，这个里面是这样解释的，就是说会自动处理下面没有列出来的其他文件，但是我还是不懂😂难道有什么优先级规则。。。
image

Comment (wendellhu95) [reply .gitattributes]: 应该是通配符有优先级，但你这些配置都一样，似乎就没有必要了

Comment (tirelyl) [reply .gitattributes]: 嗯嗯，是的，我改一下

Comment (wendellhu95) [reply .gitattributes]: .ts 没有哇

**Pull request merged**